temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityExecutionContext.java

```diff
@@ -39,6 +39,7 @@
 class LocalActivityExecutionContext {
   private final @Nonnull ExecuteLocalActivityParameters executionParams;
   private final @Nonnull AtomicInteger currentAttempt;
+  private final @Nonnull AtomicInteger attemptsDuringThisWFT;
```

Comment:
why do we need a field? just call executionParams.getAttemptsDuringThisWFT() in the getAttemptsDuringThisWFT